ICT4DNews/app/src/main/java/at/ict4d/ict4dnews/screens/splashscreen/splash.kt

```diff
@@ -0,0 +1,32 @@
+package at.ict4d.ict4dnews.screens.splashscreen
+
+import android.content.Intent
+import android.support.v7.app.AppCompatActivity
+import android.os.Bundle
+import android.os.Handler
+import at.ict4d.ict4dnews.R
+import at.ict4d.ict4dnews.screens.MainNavigationActivity
+
+class splash : AppCompatActivity() {
```

Comment:
Classes always start with a capital letter, please change to `Splash`. 
Please also don't extent from `AppCompatActivity` directly, but use our `BaseActivity`